### PR TITLE
chore: update workflows from templates

### PR DESCRIPTION
**Pull request by nextcloud-command**
This PR was automatically generated by the workflow dispatcher
Please check that there are no reverted changes before merging ⚠

See templates in https://github.com/nextcloud/.github

### CODE REVIEW

File: .github/workflows/command-compile.yml [hunk 1]
```diff
@@ -119,14 +119,14 @@ jobs:
...
-        uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v3
+        uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3
         with:
           node-version: ${{ steps.package-engines-versions.outputs.nodeVersion }}
           cache: npm
 
       - name: Set up npm ${{ steps.package-engines-versions.outputs.npmVersion }}
         run: npm i -g 'npm@${{ steps.package-engines-versions.outputs.npmVersion }}'
-
+
```
Comment (susnux) [inline .github/workflows/command-compile.yml:129]:
```suggestion

```

File: .github/workflows/command-compile.yml [hunk 2]
```diff
@@ -146,7 +146,7 @@ jobs:
         run: |
           git add '${{ github.workspace }}${{ needs.init.outputs.git_path }}'
           git commit --signoff -m 'chore(assets): Recompile assets'
-
+
```
Comment (susnux) [inline .github/workflows/command-compile.yml:149]:
```suggestion

```

File: .github/workflows/command-compile.yml (hunk 1 above)
Comment (AndyScherzinger) [reply .github/workflows/command-compile.yml:129]: rather fix on org template, no?

File: .github/workflows/command-compile.yml (hunk 2 above)
Comment (AndyScherzinger) [reply .github/workflows/command-compile.yml:149]: same as above, rather fix org template

**Comment by susnux** [comment]
Basically for server we use more customized templates so update is not that easy.

**Pull request closed without merging**